README.md

```diff
@@ -1 +1,73 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+This is a first project from microverse
+
+## Built With
+
+- npm
+- vscode
+- html
+- css
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Computer
+- Code editor
+
+### Setup
+- Clone the repository to your computer, or you download the zip file containing the repository files.
+- File editing can then be applied using any code editor.
```

Comment:
- [ ] [OPTIONAL]  I would like to recommend you add the explicit different steps to run your project locally in order to help non-technical people .  instead of saying clone the repository ... You can add the example below  :arrow_down: 
```
git clone https://github.com/PedroH-Neves/Hello-Microverse.git

Cd Hello-Microverse
```